.github/workflows/release.yml

```
@@ -470,10 +470,11 @@ jobs:
           # shellcheck disable=SC2086  # PRERELEASE_FLAG is '--prerelease' or empty
           gh release create "$RELEASE_TAG" --target "$RELEASE_COMMIT" $PRERELEASE_FLAG --title "$ANNOUNCEMENT_TITLE" --notes-file "$RUNNER_TEMP/notes.txt" artifacts/*
 
-  # Build and push Docker Hub images (:version, :latest, :sha-*) after the GitHub Release exists.
+  # Keep release-path Docker build/publish disabled for #6160. The reusable
+  # Docker workflow remains available through manual and scheduled triggers.
   docker-image:
     needs: host
-    if: ${{ always() && needs.host.result == 'success' }}
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)